src/components/TextEditor/TextEditor.vue

```diff
@@ -219,8 +215,19 @@ export default {
         this.$emit('blur', event)
       },
     })
+
+    this.$nextTick(() => {
+      if (this.$refs.editorContainer && this.editor) {
+        const tableHandler = new TableHandler(this.editor, this.$refs.editorContainer)
+        this.$refs.editorContainer.tableHandler = tableHandler
+      }
+    })
```

Comment:
this is leaky behaviour, table extension should just work if i include it in extensions list without external setup